firmware/lm32/heartbeat.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <generated/csr.h>
+#include <generated/mem.h>
+#include <hw/flags.h>
+#include <system.h>
+#include <time.h>
+
+#define HDMI_IN0_FRAMEBUFFERS_BASE 	0x01000000
+#define HDMI_IN1_FRAMEBUFFERS_BASE 	0x02000000
+#define PATTERN_FRAMEBUFFER_BASE 	0x03000000
+
+#define YCBCR422_RED    0x544cff4c
```

Comment:
Added to header file heartbeat.h 
This is actually 4bytes, corresponding to 2 bytes for Y of each pixel and average Cb and average Cr of the two pixels.